src/ensembl/src/content/app/browser/Browser.tsx

```diff
@@ -23,26 +23,28 @@ import {
 } from './browserState';
 import {
   changeBrowserLocation,
-  fetchExampleObjectsData,
-  fetchObjectData,
-  toggleDrawer,
   updateChrLocation,
   updateBrowserNavStates
 } from './browserActions';
 import {
   getBrowserOpenState,
-  getDrawerOpened,
   getBrowserNavOpened,
   getChrLocation,
   getGenomeSelectorActive,
-  getBrowserActivated,
-  getExampleObjects
+  getBrowserActivated
 } from './browserSelectors';
+import { getChrLocationFromStr, getChrLocationStr } from './browserHelper';
```

Comment:
Would it be better if we rename these functions to be `getChromosomeLocationFromStr' and `getChromsomeLocationStr`? 
Using `chr` creates a bit of confusion between chromosome and character.

Reply:
It was already long as it was so that is why I wanted to shorten it a bit. For character won't we be usually using `char` instead?

Reply:
It's just that I've come across the term `chr` being used as a shorthand for character in the past.
If `Chr` would always mean chromosome in our case, it's cool.

Reply:
The Location is not always going to be on Chromosome coordinates. depends on how good the assembly is. For example http://www.ensembl.org/Gopherus_agassizii/Location/View?r=PPEB01000001.1:490852-804510

getLocationFromStr ?

Reply:
Won't location become ambiguous and too generic when we have other kinds of locations, such as URL locations, to work with?

Reply:
getGenomeLocation...?

Reply:
That would do!

Reply:
I think this deserves to have a pull request of its own, since the change will affect a lot of files and also is not related to the Redux code.